qt/WebView.cpp

```diff
@@ -56,8 +65,6 @@
 
 std::vector<WebView*> WebView::s_instances;
 
-namespace
-{
```

Comment:
why move the following declarations out of the anonymous namespace?  those declarations appear to be fine with internal linkage?